crawl-ref/source/enum.h

```diff
@@ -2527,7 +2527,12 @@ enum monster_type                      // menv[].type
 #if TAG_MAJOR_VERSION > 34
     MONS_GIANT_LIZARD,          // genus
 #endif
+#if TAG_MAJOR_VERSION == 34
     MONS_GIANT_NEWT,
+#endif
+#if TAG_MAJOR_VERSION > 34
+    MONS_FRILLED_LIZARD,
+#endif
```

Comment:
this can be simplified (put frilled lizards right after giant lizards, use an #else for giant newts)